src/main/java/teammates/ui/template/FeedbackSessionsForm.java

```diff
@@ -131,15 +131,15 @@ public static FeedbackSessionsForm getFormForNewFs(FeedbackSessionAttributes fee
                                : SanitizationHelper.sanitizeForRichText(feedbackSession.getInstructions().getValue());
 
         newFsForm.fsStartDate = feedbackSession == null
-                              ? TimeHelper.formatDate(TimeHelper.getNextHour())
-                              : TimeHelper.formatDate(feedbackSession.getStartTimeLocal());
+                              ? TimeHelper.formatDateForSessionsForm(TimeHelper.getNextHour())
```

Comment:
Since the backend value is not used, can change this to empty string just like in line 141?

Reply:
Can try removing the entire `getNextHour` method from `TimeHelper`. Update `InstructorFeedbackSessionsPageDataTest` (expect empty string) and `HtmlHelper` (remove the `replace` statements) too. See if the build passes.